src/main/java/org/galatea/starter/utils/rest/RestResponse.java

```diff
@@ -0,0 +1,44 @@
+package org.galatea.starter.utils.rest;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Map;
+
+import org.galatea.starter.utils.Tracer;
+
+import com.google.common.collect.Maps;
+
+import lombok.Getter;
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Ensures some common audit fields for each rest response
+ */
+@Getter
+@ToString

```

Comment:
Won't fix in this pull request -- this class will go away when audit info is moved to http headers.